### ISSUE TITLE
Использование базы данных в памяти приложения

### ISSUE THREAD

**Issue by pallid** [EvilBeaver/OneScript.Web#39]
Отсылка к #38 

Сейчас для работы с данными приходится использовать удаленную базу, что накладывает сложности с транспортом и т.д. Или хранить все данные в Таблицах значений - поближе к пользователям, но ТЗ не позволяет удобно получать данные (работают только отборы по признаку равно) 

хочется иметь возможность использовать например sqlite в памяти приложения

и при инициализации указывать файл sql запроса для создания структуры БД

**Comment by EvilBeaver** [comment]
Да, это отсылка к #38 После ее починки станет доступа работа с библиотекой sql.

Активно заниматься os.web я начну с середины августа.

**Comment by pallid** [comment]
так же открыт вопрос по рабоче библиотеки sql https://github.com/ret-Phoenix/oscript-sql/issues/7 c linux

**Comment by pallid** [comment]
> Вопрос встал с миграцией модели данных

с помощью платформы? обычно же это делается отдельным приложением

или для автоматической генерации данных для миграции?

**Comment by EvilBeaver** [comment]
> или для автоматической генерации данных для миграции

Да, вопрос встал с автоматическим формированием разниц в модели старой версии и новой версии и генерацией скриптов под миграцию. В EntityFramework такое есть, кроме того есть еще FluentMigrator, но до реализации дело не дошло. Вот кто бы помог..

**Comment by pallid** [comment]
Так может это оставить на совесть тех кто делает приложение? ))) 
Сами сделали миграцию с помощью библиотеки sql и все

> Вот кто бы помог..

Сегодня только поставил VS, пока все сложно

хотел попробовать самостоятельно решить вопрос с базой в памяти

**Comment by pallid** [comment]
А модели же по идеи должны быть в os файлах, или планировался отдельный каталог в model, типа db - где все модели базы описаны?

**Comment by EvilBeaver** [comment]
А это открытый вопрос. Неизвестно
